autoreject/utils.py

```diff
@@ -108,7 +108,7 @@ def _pbar(iterable, desc, leave=True, position=None, verbose='progressbar'):
         from mne.utils import ProgressBar
 
         _ProgressBar = ProgressBar
-        if not mne.utils.check_version('mne', '0.14dev0'):
+        if mne.utils.check_version('mne', '0.14'):
```

Comment:
or let's git rid of this conditional and say in the docs that we support only the latest stable version of `mne`